azure-storage-table/info.json

```diff
@@ -0,0 +1,337 @@
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "type": "text",
+        "name": "accountName"
+      },
+      {
+        "title": "Account SAS Token",
+        "name": "sas_token",
+        "type": "password",
+        "description": "Account Shared Access Signature to perform automated operations on Storage Table Service.",
+        "tooltip": "Account Shared Access Signature to perform automated operations on Storage Table Service.",
+        "required": true,
+        "editable": true,
+        "visible": true
+      },
+      {
+        "title": "Verify SSL",
+        "required": false,
+        "editable": true,
+        "visible": true,
+        "type": "checkbox",
+        "name": "verify_ssl",
+        "value": true,
+        "description": "Specifies whether the SSL certificate for the server is to be verified or not. By default, this option is set as True."
+      }
+    ]
+  },
+  "operations": [
+    {
+      "title": "Create Table",
+      "description": "The Create Table operation creates a new table in a storage account.",
+      "operation": "create_table",
+      "category": "investigation",
+      "annotation": "create_table",
+      "parameters": [
+        {
+          "title": "Table Name",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "table_name",
+          "description": "Table name to create new table."
+        }
+      ],
+      "output_schema": {
+        "odata.metadata": "",
+        "TableName": ""
+      }
+    },
+    {
+      "title": "Delete Table",
+      "description": "The Delete Table operation deletes a table in a storage account.",
+      "operation": "delete_table",
+      "category": "investigation",
+      "annotation": "delete_table",
+      "parameters": [
+        {
+          "title": "Table Name",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "table_name",
+          "description": "Table name to delete table."
+        }
+      ],
+      "output_schema": {},
+      "enabled": true
+    },
+    {
+      "title": "Query Table",
+      "description": "The Query Tables operation returns a list of tables under the specified account.",
+      "operation": "query_table",
+      "category": "investigation",
+      "annotation": "query_table",
+      "parameters": [
+        {
+          "title": "Query Filter",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "query_filter",
+          "description": "Query expression."
+        },
+        {
+          "title": "Records Limit",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "records_limit",
+          "description": "Retrieve top n tables."
```

Comment:
Can we change the description to:
Specify the maximum number of tables that this operation should return from Azure Storage Table.